src/scse/controller/miniscot.py

```diff
@@ -118,8 +123,20 @@ def get_initial_env_values(self):
                 if module_state:
                     state[module.get_name()] = module_state
 
+        # penalize the use of many batteries
+        # penalty scaled by capacity of battery
+        # TODO: check that this makes sense
+        battery_penalty = 1
```

Comment:
@collinskatie Could we move this to the `reset` method in `src/scse/metrics/national_grid_cash_accounting.py`? Would mean that all of the reward function logic is in one place, and avoids having anything profile-specific in the core logic.

Reply:
Completed! Great suggestion -- see commit "5a9ff8b"